### ISSUE TITLE
kmod-usb-net-ipheth

### ISSUE THREAD

**Issue by mardosas** [Firmware-routers/Easybox-904-xDSL#1]
Hi,

I would to install the package ‘kmod-usb-net-ipheth’ which is included in the repository

src/gz easybox904_core https://github.com/Firmware-routers/Easybox-904-xDSL/tree/main/Repository/EB904xDSL/VPE/v1/core

kmod-usb-net-ipheth_5.4.203-1_mips_24kc.ipk

However, after running opkg update (list of packages download and updated sucessfully), opkg cannot install the package and throws an error message:

root@EasyBOX: ~$ opkg install kmod-usb-net-ipheth
Unknown package 'kmod-usb-net-ipheth'.
Collected errors:
 * opkg_install_cmd: Cannot install package kmod-usb-net-ipheth.

Exactly the same is happening when I am trying to install asterisk-chan-lantiq

root@EasyBOX: ~$ opkg install asterisk-chan-lantiq
Installing asterisk-chan-lantiq (2021-09-11-2f029ec8-1) to root...
Collected errors:
 * opkg_download_pkg: Package asterisk-chan-lantiq is not available from any configured src.
 * opkg_install_pkg: Failed to download asterisk-chan-lantiq. Perhaps you need to run 'opkg update'?
 * opkg_install_cmd: Cannot install package asterisk-chan-lantiq.

**Comment by zuzia-dev** [comment]
Hi!
Thank you for reporting. This is a bug. I will correct in the next firmware versions. 
Run the commands in the shell via ssh:
```
cd /etc/opkg
rm customfeeds.conf
wget https://raw.githubusercontent.com/Firmware-routers/Easybox-904-xDSL/main/Repository/EB904xDSL/VPE/v1/customfeeds.conf
opkg update
opkg install kmod-usb-net-ipheth
```

**Comment by mardosas** [comment]
Hi,
Thank you for the updated customfeeds.conf
However, opkg throws an error:
Collected errors:
 * opkg_download: Failed to download https://raw.githubusercontent.com/Firmware-routers/repo-test/main/VPE/v1/core/Packages.gz, wget returned 8.
 * opkg_download: Failed to download https://raw.githubusercontent.com/Firmware-routers/repo-test/main/VPE/v1/easybox904/Packages.gz, wget returned 8.

Both Packages.gz seems not to be available for download.

**Comment by zuzia-dev** [comment]
Hi @mardosas 
Update: The problem has been resolved in the new version of OpenWrt 21.02.7:
VPE: https://github.com/zuzia-dev/Easybox-904xDSL-repo-source/blob/main/Firmware/VPE/v2/openwrt-lantiq-xrx200-arcadyan_vgv952cjw33-e-ir-vpe-squashfs-sysupgrade.bin
SMP: https://github.com/zuzia-dev/Easybox-904xDSL-repo-source/blob/main/Firmware/SMP/v2/openwrt-lantiq-xrx200-arcadyan_vgv952cjw33-e-ir-smp-squashfs-sysupgrade.bin
